### PR TITLE
Removes outdated 'datetime/overview' data from benchmark data.

### LINKED ISSUE #673: Benchmark dashboard keeps showing inactive benchmark
The first benchmark in https://unicode-org.github.io/icu4x-docs/benchmarks/perf/components/datetime/ has been split into two and the top one is now inactive. Can we remove it?

### PR DESCRIPTION
**Pull request by gnrunge**
The original 'datetime/overview' benchmark has been split in two.

Fixes #673.

### CODE REVIEW

**Review by echeran — APPROVED** [review]
LGTM

**Comment by echeran** [comment]
Merged on behalf of @gnrunge . Don't know why Github is not allowing the PR to be linked to the issue

**Review by gregtatum — COMMENTED** [review]
LGTM (looking after merge)

**Pull request merged**
